ui/app/templates/vault/cluster/policies.hbs

```diff
@@ -1,44 +1,32 @@
-<div class="columns">
-  {{#menu-sidebar title="Policies" class="is-3" data-test-sidebar=true}}
-    {{#if (has-permission "policies" routeParams="acl")}}
-      <li>
-        {{#link-to "vault.cluster.policies" "acl" data-test-link=true class=(if (is-active-route "vault.cluster.policies" "acl") "is-active")}}
-          ACL Policies
-        {{/link-to}}
-      </li>
-    {{/if}}
-    {{#if (has-permission "policies" routeParams="rgp")}}
-      <li>
-        {{#link-to "vault.cluster.policies" "rgp" data-test-link=true class=(if (is-active-route "vault.cluster.policies" "rgp") "is-active")}}
-          Role Governing Policies
-
-        {{#unless (has-feature "Sentinel")}}
-          {{#if (is-version "OSS")}}
-            {{edition-badge edition="Enterprise"}}
-          {{else}}
-            {{edition-badge edition="Premium"}}
-          {{/if}}
-        {{/unless}}
-        {{/link-to}}
-      </li>
-    {{/if}}
-    {{#if (has-permission "policies" routeParams="egp")}}
-      <li>
-        {{#link-to "vault.cluster.policies" "egp" data-test-link=true class=(if (is-active-route "vault.cluster.policies" "egp") "is-active")}}
-          Endpoint Governing Policies
-
-        {{#unless (has-feature "Sentinel")}}
-          {{#if (is-version "OSS")}}
-            {{edition-badge edition="Enterprise"}}
-          {{else}}
-            {{edition-badge edition="Premium"}}
-          {{/if}}
-        {{/unless}}
-        {{/link-to}}
-      </li>
-    {{/if}}
-  {{/menu-sidebar}}
-  <div class="column is-9">
-    {{outlet}}
+{{#if (and (has-feature "Sentinel") (or (has-permission "policies" routeParams="rgp") (has-permission "policies" routeParams="egp")))}}
```

Comment:
should the sidebar be visible if sentinel is enabled and the user has access to `sys/policies/acl`? if so, you can update this conditional to only call `has-permission "policies"` once. this is because calling `has-permission` without specifying `routeParams` calls `hasNavPermission` under the hood, which checks if the user has access to at least one, but not all, of the paths associated with each nav item. here's the associated list of paths that the service checks for policies:

https://github.com/hashicorp/vault/blob/master/ui/app/services/permissions.js#L13-L16

Reply:
ha, nevermind. just re-read your description above. my bad!

Reply:
If all they have access to (whether by license or permissions) is ACL, then I don't want it to show the sidebar. That will simplify things for OSS users, who would only see ACL in the sidebar, which they would be getting to through the top nav "Policies" link anyway.

Does that sound right or is there a better way to do that? Should the conditional be moved to the controller anyway?